.golangci.yaml

```diff
@@ -55,8 +54,12 @@ linters-settings:
         alias: iotools
       - pkg: github.com/kyma-project/modulectl/internal/common/errors
         alias: commonerrors
-      - pkg: github.com/kyma-project/modulectl/cmd/modulectl/create/scaffold
+      - pkg: github.com/kyma-project/modulectl/cmd/modulectl/scaffold
         alias: scaffoldcmd
+      - pkg: github.com/kyma-project/modulectl/cmd/modulectl/create
+        alias: createcmd
+      - pkg: github.com/kyma-project/modulectl/internal/common/utils/test
```

Comment:
As previously mentioned: Can you move these into /internal/testutils/ please

Reply:
Also makes this alias whitelist obsolete I think

Reply:
I think in the previous review I had an option to choose between `/internal/common/utils/test/` and `/testutils/`, so I went with the former. But okay, I don't mind changing it to `/internal/testutils/` either, done.

Reply:
Yes it also makes the alias whitelisting obsolete, so that's removed now.